keyboards/crkbd/keymaps/hvp/keymap.c

```diff
@@ -0,0 +1,326 @@
+     KC_TAB,  KC_1,  KC_2,  KC_3,  KC_4,  KC_5,                   KC_6,  KC_7,  KC_8,  KC_9,  KC_0,KC_BSPC,\
+  //|------+------+------+------+------+------|                |------+------+------+------+------+------|
+    KC_ESC, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,                KC_LEFT,KC_DOWN,KC_UP, KC_RIGHT,KC_NO, KC_DEL,\
+  //|------+------+------+------+------+------|                |------+------+------+------+------+------|
+    KC_LCTL, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,                  KC_HOME, KC_PGDN, KC_PGUP, KC_END, KC_NO, KC_NO,\
+  //|------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+                                KC_LGUI, LOWER,MT(MOD_LSFT, KC_SPC),   LT(_RAISE,KC_ENT), RAISE,KC_LALT \
+                              //`--------------------'  `--------------------'
+  ),
+
+  [_LOWER] = LAYOUT( \
+  //,-----------------------------------------.                ,-----------------------------------------.
+     KC_TAB,KC_EXLM,KC_AT,KC_HASH,KC_DLR,KC_PERC,              KC_CIRC,KC_AMPR,KC_ASTR,KC_LPRN,KC_RPRN,KC_BSPC,\
+  //|------+------+------+------+------+------|                |------+------+------+------+------+------|
+    KC_ESC, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,                KC_MINS,KC_EQL,KC_LCBR,KC_RCBR,KC_PIPE,KC_GRV,\
+  //|------+------+------+------+------+------|                |------+------+------+------+------+------|
+    KC_LCTL, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,                KC_UNDS,KC_PLUS,KC_LBRC,KC_RBRC,KC_BSLS,KC_TILD,\
+  //|------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+                                KC_LGUI, LOWER,MT(MOD_LSFT, KC_SPC),   LT(_RAISE,KC_ENT), RAISE,KC_LALT \
+                              //`--------------------'  `--------------------'
+  ),
+
+  [_ADJUST] = LAYOUT( \
+  //,-----------------------------------------.                ,-----------------------------------------.
+      KC_F1,KC_F2, KC_F3, KC_F4, KC_F5, KC_F6,                  KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12,\
+  //|------+------+------+------+------+------|                |------+------+------+------+------+------|
+    KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,                 KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,\
+  //|------+------+------+------+------+------|                |------+------+------+------+------+------|
+    RESET,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,                 KC_MPLY, KC_MNXT, KC_LEFT, KC_DOWN, KC_UP, KC_RIGHT,\
+  //|------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+                                KC_LGUI, LOWER,MT(MOD_LSFT, KC_SPC),   LT(_RAISE,KC_ENT), RAISE,KC_LALT \
+                              //`--------------------'  `--------------------'
+  )
+};
+
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
@drashna  Not sure how/ what to remove here. Do you have any more references?